docker/sandbox/stubs/mappings/routing-1.json

```diff
@@ -1,7 +1,7 @@
 {
   "request": {
     "method": "GET",
-    "url": "/routing"
+    "urlPathPattern": "/routing.*"
```

Comment:
wildcard to match arguments like in example:
"/routing?org-code=YES&service-id=urn:nhs:names:services:pdsquery:QUPA_IN040000UK32"